### PR TITLE
Fix MongoDB example

### PR DESCRIPTION
**Pull request by tuxdna**
Following changes have been made:
- Update MONGODB_USER env variable. Add YAML definition for Nulecule
- Update Nulecule to YML format for consistency across other examples
- Update Docker artifiact.
- Update README file.

### CODE REVIEW

File: mongodb-centos7-atomicapp/README.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ It's a single container application based on the ``kolab/mongodb`` image.
```
Comment (dustymabe) [inline mongodb-centos7-atomicapp/README.md:6]: still a reference to kolab above ^^

File: mongodb-centos7-atomicapp/Nulecule [hunk 2]
```diff
@@ -1,68 +1,39 @@
...
+        default: true
+      - name: mongodb_password
+        description: Password for the $mongodb_user account
+      - name: mongodb_quiet
+        description: Run MongoDB quietly.
+        default: true
+      - name: mongodb_smallfiles
+        description: Configure MongoDB to use a smaller default data file size
+        default: true
+      - name: mongodb_username
```
Comment (cdrage) [inline mongodb-centos7-atomicapp/Nulecule:29]: this should be mongodb_user

File: mongodb-centos7-atomicapp/artifacts/docker/run-mongodb-atomicapp-pod [hunk 3]
```diff
@@ -1,11 +1 @@
...
-    -e MONGODB_ADMIN_PASSWORD=$mongodb_admin_password \
-    -e MONGODB_DATABASE=$mongodb_database \
-    -e MONGODB_NOPREALLOC=$mongodb_noprealloc \
-    -e MONGODB_PASSWORD=$mongodb_password \
-    -e MONGODB_QUIET=$mongodb_quiet \
-    -e MONGODB_SMALLFILES=$mongodb_smallfiles \
-    -e MONGODB_USERNAME=$mongodb_username \
-    -p 27017:27017 \
-    $image
+docker run -d --name mongodb-atomicapp-app -e MONGODB_ADMIN_PASSWORD=$mongodb_admin_password -e MONGODB_DATABASE=$mongodb_database -e MONGODB_NOPREALLOC=$mongodb_noprealloc -e MONGODB_PASSWORD=$mongodb_password -e MONGODB_QUIET=$mongodb_quiet -e MONGODB_SMALLFILES=$mongodb_smallfiles -e MONGODB_USER=$mongodb_username -p 27017:27017 $image
```
Comment (cdrage) [inline mongodb-centos7-atomicapp/artifacts/docker/run-mongodb-atomicapp-pod:1]: mongodb_user

**Comment by cdrage** [comment]
Hey @tuxdna so as much as we dislike JSON, we're going to keep that in to have a diversity of examples. I've updated my PR with your changes listed here. Thanks for the PR :)

**Comment by dustymabe** [comment]
@cdrage @tuxdna right. It's good to have examples there for our users to view as examples. Getting rid of this example might mean we have to create a new JSON example sometime and I certainly don't want to do that :)

**Pull request closed without merging**